src/detection/card/content/wrapper/box.jsx

```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+import { Box as MaterialBox } from '@material-ui/core'
+import PropTypes from 'prop-types'
+
+import { useStyles } from 'detection/card/content/wrapper/box-style'
+
+export const Box = ({ children }) => <MaterialBox classes={useStyles()}>
```

Comment:
![image](https://user-images.githubusercontent.com/23569938/125787317-e19a6b3c-0248-463b-b374-2cba72536c52.png)
Let's rename these two to something else, as their names can lead to confusion.